module-1/dist/assets/css/style-contacts.css

```diff
@@ -433,6 +80,12 @@ dropdown.closed {
   resize: none;
 }
 
+.color__placeholder::placeholder {
+  font-size: 19px;
+  color: #f78707;
```

Comment:
Don't use colours directly, move all colours to a variables in a separate file, call it `_colors.scss` and use it this way. Also why we using plain css here? It must be `SASS`, nobody uses vanilla css anymore. Please discuss with @USergei how to fix build scripts that were originally using SASS